app/src/lib/stores/app-store.ts

```diff
@@ -504,16 +500,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
       error => this.emitError(error)
     )
 
-    const browserWindow = remote.getCurrentWindow()
-    this.windowState = getWindowState(browserWindow)
-
-    this.onWindowZoomFactorChanged(browserWindow.webContents.zoomFactor)
```

Comment:
Why isn't this necessary any more? I would think we'd have to get a hold of the zoomFactor before it changes (like if the app starts with a zoomFactor other than 1)